packages/theme/src/theme.ts

```diff
@@ -131,7 +133,7 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
             }
             return acc;
         }, [] as CSSResult[]);
-        return [coreStyles, ...styles];
+        return [...styles];
```

Comment:
Was this an intended change?

Reply:
Yep. I was doing a test of the set and realized we were adding that style twice 🙈 so the logic actually made that length that we test whether to adopt the styles or not work different than intended.